From 3ac819e2a6b798c42adae6f32e1259d9011833e2 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Fri, 16 Sep 2022 13:23:19 +0300
Subject: [PATCH] HACK: arm64: dts: v2x: disable lte vbat regulator

Work-around for particular power-sequence requirements of LTE modem.
Allows toggling all of reset, pwrbutton and power GPIOs from userspace.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 arch/arm64/boot/dts/freescale/imx8dxl-v2x.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8dxl-v2x.dts b/arch/arm64/boot/dts/freescale/imx8dxl-v2x.dts
index dfe3809a3039..e6248af885e0 100644
--- a/arch/arm64/boot/dts/freescale/imx8dxl-v2x.dts
+++ b/arch/arm64/boot/dts/freescale/imx8dxl-v2x.dts
@@ -13,7 +13,7 @@ / {
 	model = "SolidRun i.MX8DXL V2X Carrier";
 	compatible = "fsl,imx8dxl-sr-som", "fsl,imx8dxl";
 
-	lte_vbat_reg: regulator-lte-vbat {
+	/*lte_vbat_reg: regulator-lte-vbat {
 		compatible = "regulator-fixed";
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_lte_vbat>;
@@ -23,7 +23,7 @@ lte_vbat_reg: regulator-lte-vbat {
 		gpio = <&lsio_gpio0 14 GPIO_ACTIVE_HIGH>;
 		enable-active-high;
 		regulator-always-on;
-	};
+	};*/
 
 	regulator_1v1: regulator-1v1 {
 		compatible = "regulator-fixed";
-- 
2.38.1

